docs/user/03-21-instance-level-mapping.md

```diff
@@ -1,21 +1,22 @@
-# Create a Service Instance with a Custom Secret
+# Instance-Level Mapping
```

Comment:
What is the meaning of the title?

Reply:
I've added the explanation you provided - thank you :)